pkgs/applications/networking/remote/remmina/default.nix

```diff
@@ -56,6 +56,7 @@ stdenv.mkDerivation rec {
 
   preFixup = ''
     gappsWrapperArgs+=(
+      --set SSL_CERT_DIR "/etc/ssl/certs/"
```

Comment:
Is there a use-case for overriding this without having to rebuild the derivation? If so, we can use`--set-default` instead of `--set`.

Reply:
I just had a small concern about setting the variable hard (ignoring the existing environment). An alternative would be --set-default which would use the value from the enviroment, but if missing, use the given/fallback value.

Reply:
Thank you for the clear and patient explanation. I agree and I changed it to `--set-default`